djlsp/server.py

```diff
@@ -94,6 +98,9 @@ def set_initialization_options(self, options: dict):
         self.django_settings_module = options.get(
             "django_settings_module", self.django_settings_module
         )
+        self.cache = options.get("cache", self.cache)
+        if self.cache is True:
+            self.cache = os.path.join(tempfile.gettempdir(), "djlsp-data.json")
```

Comment:
If we use os tempdir location as default location, the workspace location should be in name to support multiple projects (this can be hash of path).